services/client/services.go

```diff
@@ -397,7 +412,15 @@ func (c *client) watchPlacementAndHeartbeat(
 			if value == nil {
 				// NB(cw) this can only happen when the placement has been deleted
 				// it is safer to let the user keep using the old topology
-				c.logger.Infof("received placement update with nil value")
```

Comment:
This looks the same as the logic above in `watchPlacement`. Can we move them into a method so for future changes we don't need to make the same change in two places?